listings/ch00-introduction/maps/Scarb.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "maps"
+version = "0.1.0"
+
+# See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest
+
+[dependencies]
+starknet = "1.1.0"
+
+[[target.starknet-contract]]
+allowed-libfuncs-list.name = "experimental_v0.1.0"
```

Comment:
Can you remove this line? It's used for experimental libfuncs only that are not whitelisted for mainnet yet - but in this example, there is no need for that